site2/docs/io-quickstart.md

```diff
@@ -378,11 +338,415 @@ cqlsh:pulsar_test_keyspace> select * from pulsar_test_table;
+
+```
+$ bin/pulsar-admin sink list \
+--tenant public \
+--namespace default
+```
+
+The result shows that only the _mysql-jdbc-sink_ sink is running.
+
+```
+[
+ "pulsar-mysql-jdbc-sink"
+]
+```
+
+### Get information of a JDBC sink
+
+This example gets the information about the _pulsar-mysql-jdbc-sink_ sink connector.
+
+```
+$ bin/pulsar-admin sink get \
+--tenant public \
+--namespace default \
+--name pulsar-mysql-jdbc-sink
+```
+
+The result show the information of the sink connector, including tenant, namespace, topic and so on.
+
+```
+{
+  "tenant": "public",
+  "namespace": "default",
+  "name": "pulsar-mysql-jdbc-sink",
+  "className": "org.apache.pulsar.io.jdbc.JdbcAutoSchemaSink",
+  "inputSpecs": {
+    "pulsar-mysql-jdbc-sink-topic": {
+      "isRegexPattern": false
+    }
+  },
+  "configs": {
+    "password": "jdbc",
+    "jdbcUrl": "jdbc:mysql://127.0.0.1:3306/pulsar_mysql_jdbc_sink",
+    "userName": "root",
+    "tableName": "pulsar_mysql_jdbc_sink"
+  },
+  "parallelism": 1,
+  "processingGuarantees": "ATLEAST_ONCE",
+  "retainOrdering": false,
+  "autoAck": true
+}
+```
+
+### Get status of a JDBC sink
+
+This example checks the current status of the _pulsar-mysql-jdbc-sink_ sink connector.
+
+```
+$ bin/pulsar-admin sink status \
+--tenant public \
+--namespace default \
+--name pulsar-mysql-jdbc-sink
+```
+
+The result shows the current status of sink connector, including the number of instance, running status, worker ID and so on.
+
+```
+{
+  "numInstances" : 1,
+  "numRunning" : 1,
+  "instances" : [ {
+    "instanceId" : 0,
+    "status" : {
+      "running" : true,
+      "error" : "",
+      "numRestarts" : 0,
+      "numReadFromPulsar" : 0,
+      "numSystemExceptions" : 0,
+      "latestSystemExceptions" : [ ],
+      "numSinkExceptions" : 0,
+      "latestSinkExceptions" : [ ],
+      "numWrittenToSink" : 0,
+      "lastReceivedTime" : 0,
+      "workerId" : "c-standalone-fw-192.168.2.52-8080"
+    }
+  } ]
+}
+```
+
+## Stop a JDBC sink
+
+This example stops the _pulsar-mysql-jdbc-sink_ sink instance.
+
+```
+$ bin/pulsar-admin sink stop \
+--tenant public \
+--namespace default \
+--name pulsar-mysql-jdbc-sink \
+--instance-id 0
+```
+
+The sink instance has been stopped successfully if the following message disappears.
+
+```
+"Stopped successfully"
+```
+
+## Restart a JDBC sink
+
+This example starts the _pulsar-mysql-jdbc-sink_ sink instance.
+
+```
+$ bin/pulsar-admin sink start \
+--tenant public \
+--namespace default \
+--name pulsar-mysql-jdbc-sink \
+--instance-id 0
+```
+
+The sink instance has been started successfully if the following message disappears.
+
+```
+"Started successfully"
+```
+
+> ### Tip
+> Optionally, you can run a standalone sink connector using `pulsar-admin sink localrun options`. 
+> 
+> Note that `pulsar-admin sink localrun options` runs a sink connector locally, while `pulsar-admin sink start options` can run a sink connector locally or in a cluster.
```

Comment:
I think the statement here is a bit confused. `sink start` will start a connector in a cluster, not run a connector locally, right?

Reply:
Thanks, I've changed it to:
 `pulsar-admin sink start options` starts a connector in a cluster.